### PR TITLE
feat: add chinese trad for taiwan translations

### PR DESCRIPTION
**Pull request by tso1158687**
feat(lang): add chinese traditional language for Taiwanese

Type: translation

The following has been addressed in the PR:

There is a related issue?
NO
Unit or Functional tests are included in the PR
NO
Description: Add Chinese Traditional translations for Taiwanese (zh-TW)

### CODE REVIEW

**Comment by juanpicado** [comment]
> feat(lang): add chinese traditional language for Taiwanese
> 
> 
> 
> Type: translation
> 
> 
> 
> The following has been addressed in the PR:
> 
> 
> 
> There is a related issue?
> 
> NO
> 
> Unit or Functional tests are included in the PR
> 
> NO
> 
> Description: Add Chinese Traditional translations for Taiwanese (zh-TW)

Awesome !!! Thanks 😊 (btw I love Taiwan, my last visit was priceless)

**Comment by juanpicado** [comment]
@tso1158687 could you rebase? I just merged a new feature wich conflict with your code.

**Comment by tso1158687** [comment]
> btw I love Taiwan, my last visit was priceless

Thanks for loving Taiwan 😄

**Comment by tso1158687** [comment]
> @tso1158687 could you rebase? I just merged a new feature wich conflict with your code.

Sure, I'll rebase it

File: src/components/LanguageSwitch/LanguageSwitch.tsx [hunk 1]
```diff
@@ -102,10 +106,8 @@ const LanguageSwitch = () => {
     ),
     [getCurrentLngDetails]
   );
-
-  const optionLabel = useCallback(({ translation }: ReturnType<typeof getCurrentLngDetails>) => translation, [
-    getCurrentLngDetails,
-  ]);
+  // prettier-ignore
```
Comment (priscilawebdev) [inline src/components/LanguageSwitch/LanguageSwitch.tsx:109]: Please why did you add 'prettier-ignore' here?

**Comment by priscilawebdev** [comment]
@tso1158687  Good Job! Thank you!.. if you don't mind I will update the snapshots, so we can merge it 😉🙂

File: src/components/LanguageSwitch/LanguageSwitch.tsx (hunk 1 above)
Comment (tso1158687) [reply src/components/LanguageSwitch/LanguageSwitch.tsx]: I can not pass lint check because of this line, but I don't know how to fix it.
So I add prettier-ignore to try but I forget removing this.
Sorry for confused you

**Comment by tso1158687** [comment]
> @tso1158687 Good Job! Thank you!.. if you don't mind I will update the snapshots, so we can merge it 😉🙂

Thank you!
